.github/workflows/perform-release.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           # Wait for the deploy-artifacts.yml workflow to start
           sleep 3
           # Get the URL of the release workflow (MUST match the workflow's run-name)
-          SUMMARY_URL=$(gh run list --workflow deploy-artifacts.yml -R quarkiverse/quarkiverse-release --json name,url -q ".[] | select(.name == \"${WORKFLOW_RELEASE_RUN_NAME}\") | .url" --limit 1)
+          SUMMARY_URL=$(gh run list --workflow deploy-artifacts.yml -R smallrye/smallrye-release --json name,url -q ".[] | select(.name == \"${WORKFLOW_RELEASE_RUN_NAME}\") | .url" --limit 1)
```

Comment:
Do you have an example somewhere? 

How does it work if the process is not done yet.

Reply:
This returns the queued workflow run URL, it doesn’t matter its state, you can monitor the logs related to this deployment once you click on it